docs/getting-started.md

```diff
@@ -76,6 +76,16 @@ An Azure Service Principal is needed for populating the controller manifests. Th
   ```
 <!--An alternative is to install [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) and have the project's script create the service principal automatically. _Note that the service principals created by the scripts will not be deleted automatically._ -->
 
+  5. SSH keys
+
+  By default a new ssh key is generated, to setup your own ssh key, set the `AZURE_SSH_PUBLIC_KEY_FILE` environment variable
```

Comment:
this only applies to E2E tests, not general manual deployments and this documentation is for users, not developers using E2E.

Reply:
Ok, will move to another place

Reply:
looks like there is a documentation already